docs/topics/proc_creating-an-application.adoc

```diff
@@ -22,7 +22,7 @@ The following procedure shows you how to:
 .Prerequisites
 
 * Maven installed.
-* JDK 8 or later installed.
+* {prerequisite-jdk-version}
```

Comment:
I think this is not the case with Thorntail 2.2.0, so we can't make this replacement. @huntley, please have a look into this, thanks.

Reply:
@tradej, we can conditionalize it to show a Java 8 JDK only for TT. WDYT?